### ISSUE TITLE
Include support for Gradle and Maven 

### ISSUE THREAD

**Issue by DarkerMinecraft** [Mudbill/dds-lwjgl#7]
Just a quick suggestion on making it available on Gradle and Maven platforms. You can check at https://jitpack.io but you only need a build file I believe.

**Comment by Mudbill** [comment]
I've never actually worked with either. If you think it's worth adding, you can submit a PR.

**Comment by nimelord** [comment]
I will do if You get me permissions.
Actually I need your lib as a maven artifact.

**Comment by Mudbill** [comment]
I will look into publishing it on Maven central.

**Comment by Mudbill** [comment]
Okay, so I think I have set it all up now. Give it a try and let me know if everything works :)

https://central.sonatype.com/artifact/io.github.mudbill/dds-lwjgl

For Maven:
```xml
<dependency>
    <groupId>io.github.mudbill</groupId>
    <artifactId>dds-lwjgl</artifactId>
    <version>2.1.1</version>
</dependency>
```

I might consider moving the package namespace within to match the github namespace I used for the artifact. Although I technically own both, I just found this to be easier.

**Comment by nimelord** [comment]
Unfortunately I can't use this build.
It downloaded well, but:
`[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project zengine: Compilation failure: Compilation failure: 
[ERROR] /data/projects/zingl/zengine/src/main/java/com/zingl/engine/graph/texture/ImageDecodeUtils.java:[8,31] cannot access net.buttology.lwjgl.dds.DDSFile
[ERROR]   bad class file: /home/johny/.m2/repository/io/github/mudbill/dds-lwjgl/2.1.1/dds-lwjgl-2.1.1.jar(/net/buttology/lwjgl/dds/DDSFile.class)
[ERROR]     class file has wrong version 65.0, should be 61.0
`

That means You use 21 version of jdk to build lib. It is pretty modern and so many platforms still use 17th.)

**Comment by Mudbill** [comment]
Alright, I will look to fix this shortly. Just an oversight on my part – the code should compile fine for JDK 8.
Thanks for letting me know.

**Comment by Mudbill** [comment]
I've published a new version: 3.0.0.
This one should be compatible all the way back to JDK 1.8, so that should fix those issues. Give it a try and let me know 😄 

I also optimized some of the dependencies and such. I decided to go with LWJGL version 3.3.0, although it's possible it's supported further back. I'd have to test that.

However I made 1 **breaking change** (hence the major version): I updated the package to better reflect its component artifact on Central. Instead of importing `net.buttology.lwjgl.dds.DDSFile`, you must update it to import `io.github.mudbill.dds.DDSFile`. A simple change, but avoids potential confusion going forward.
